firmware/projects/Demo/CANErrors/README.md

```diff
@@ -1,13 +1,36 @@
 # CANErrors
+This is the CAN Errors demo project. It defines a global system for the vehicle to send/receive errors through different subsystems. It defines:
+- A 64 bit CAN error signal. Each bit represents a different error, and will be sent periodically for other systems to know if an error has occurred
+- The `ErrorHandler` class, an interface to set/send different errors on the CAN signal
+- Enum `Error` that defines all 64 possible errors to send through the CAN signal
 
-This is the CAN Errors demo project. This defines the mechanism for reporting errors from the vehicle microcontrollers. This project will send errors over the CAN line and process them on a host machine.
+<br>
```

Comment:
this is a bit pedantic but stick to the regular markdown formatting i.e. accept the default paragraph spacing instead of adding a `<br>`